### DISCUSSION TITLE
Kali linux support

### DISCUSSION THREAD

**Discussion by sudip-mondal-2002** [community/community#37065] [Codespaces] [answered]
I couldn't find any default Kali linux support on codespaces. Is there any way around? Thanks in advance :)

**Comment by ossanna16** [comment]
Hi there @sudip-mondal-2002 and welcome to our community 😄. Thanks so much for asking a great question!

**Comment by cmbrose** [comment, marked as answer]
Hi @sudip-mondal-2002 - thanks for the question! You can set up your codespace to use the Kali linux image in a few ways, the quickest is to create a `.devcontainer.json` file in the repo root and point it at the image, like so:

```json
{
  "image": "kalilinux/kali-rolling:latest"
}
```

You can do a lot more with devcontainers as well to help with installations such as adding some post creations scripts, or creating a full `DOCKERFILE` for it as well. 

You can read more about that in the docs [here](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers) and I'd be happy to answer further specific questions. Thanks!

**Comment by Sachinydv001** [comment]
Can anyone help me to reset my kali password because i forgot it

**Reply by ghost** [reply]
no

**Reply by js20-web** [reply]
Why did you say that?
